shell/ci/release/docker.sh

```diff
@@ -34,7 +34,7 @@ get_image_field() {
   local name="$1"
   local field="$2"
   # shellcheck disable=SC1087 # this is a yq/jq filter that shellcheck thinks is bash
-  yq -r ".[\"$name\"]$field[]" "$MANIFEST"
+  yq -r ".[\"$name\"]$field[]?" "$MANIFEST"
```

Comment:
TIL, I would've just checked the field type. This is much cooler. You might want to test it with the instructions in the README on a repo w/ and w/o the settings to be safe.